command/src/config.rs

```diff
@@ -232,16 +309,20 @@ impl Listener {
             address: self.address,
             sticky_name: self.sticky_name.clone(),
             public_address: self.public_address,
+            tls_provider: self.tls_provider.clone(),
             cipher_list,
+            cipher_suites,
+            signature_algorithms,
+            groups_list,
             versions,
             expect_proxy,
-            rustls_cipher_list,
             key,
             certificate,
             certificate_chain,
             front_timeout: self.front_timeout.or(front_timeout).unwrap_or(60),
             back_timeout: self.back_timeout.or(back_timeout).unwrap_or(30),
             connect_timeout: self.connect_timeout.or(connect_timeout).unwrap_or(3),
+            request_timeout: 0,
```

Comment:
I see you're instantiating with a request_timeout of 0, instead of the default 10. Is there a particular reason?

This reminds me of issue #741 that shows that `request_timeout` is not propagated from `bin/config.toml`, which is true and which I have to fix someday.

Just so you know.